ctx-module.js

```diff
@@ -547,7 +547,11 @@ exports.makeNodeProgramContext = function makeNodeProgramContext(options)
 function copyProps(dst, src)
 {
   const pds = Object.getOwnPropertyDescriptors(src);
-  Object.defineProperties(dst, pds);
+  for (var propds in pds)
```

Comment:
Nitpick/Style: use `const` instead of `var`
```suggestion
  for (const propds in pds)
```

https://github.com/Distributive-Network/eslint-config?tab=readme-ov-file#iteration

Reply:
This is not nit, `var` and `function` variables are prohibited as loop invariants in our style.  We must either `let` or `const` to get a fresh binding for each iteration of the loop.

In this case it probably doesn't have an observable side effect, but it easily could throughout the maintenance cycle.